packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -554,95 +536,6 @@ void main() {
     }, overrides: <Type, Generator>{
       DeviceManager: () => mockDeviceManager,
     });
-
-    group('--dart-define option', () {
```

Comment:
this is tested via various devicelab and integration tests, the unit tests has too many mocks to be useful and was failing with some wicked stack traces